pkgs/ffigen/lib/src/visitor/visitor.dart

```diff
@@ -2,24 +2,34 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:logging/logging.dart';
+
 import '../code_generator.dart';
 
 import 'ast.dart';
 
+final _logger = Logger('ffigen.visitor');
+
 /// Wrapper around [Visitation] to be used by callers.
 final class Visitor {
-  Visitor(this._visitation) {
+  Visitor(this._visitation, {bool debug = false}) : _debug = debug {
     _visitation.visitor = this;
   }
 
   final Visitation _visitation;
   final _seen = <AstNode>{};
+  final bool _debug;
+  int _ind = 0;
```

Comment:
nit: better to write the full `_indentLevel`, I initially read it as `_index`

Reply:
Done